src/app/views/templates/payment-confirmation.view.html

```diff
@@ -1,12 +1,124 @@
 <!DOCTYPE html>
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
-  <body>
-    <p>Hello there,</p>
-    <p>
-      Your payment on <%= appName %> form: <%= formTitle %> has been received
-      successfully. Your response ID is <%= submissionId %> and your proof of
-      payment can be found <a href="<%= invoiceUrl %>">here</a>.
-    </p>
-    <p>Regards,<br /><%= appName %> team</p>
+  <head>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap');
+      body {
+        font-family: 'Inter', sans-serif;
+      }
+    </style>
+  </head>
+  <body
+    class="clean-body u_body"
+    style="margin: 0; padding: 0; background-color: #f8f9fd"
+  >
+    <div
+      class="u-row-container"
+      style="
+        padding: 48px;
+        background-color: #ffffff;
+        width: 507px;
+        margin-left: auto;
+        margin-right: auto;
+        margin-bottom: auto;
+        margin-top: 48px;
+      "
+    >
+      <p>
+        <img width="94.14px" src="https://file.go.gov.sg/form-logo.png" />
+      </p>
+      <p
+        style="
+          font-style: normal;
+          font-weight: 600;
+          font-size: 24px;
+          line-height: 32px;
+          letter-spacing: -0.019em;
+          font-feature-settings: 'cv05' on, 'cv10' on;
+
+          color: #1b1f2c;
+          margin-bottom: 32px;
+        "
+      >
+        Your payment on <%= formTitle %> has been received successfully.
+      </p>
+      <div
+        style="
+          font-style: normal;
+          font-weight: 400;
+          font-size: 16px;
+          line-height: 24px;
+          letter-spacing: -0.006em;
+
+          color: #474747;
+
+          margin-bottom: 32px;
+        "
+      >
+        <p style="margin: 0px">Form: <%= formTitle %></p>
```

Comment:
Looking at the figma, it seems like the inputs `formTitle`, `submissionId`, etc are bolded (larger font-weight size)

Reply:
Thanks for catching this @wanlingt have changed it here [a31fff9](https://github.com/opengovsg/FormSG/pull/6704/commits/a31fff9352e0dce9e377d145e79afa8b69dd156b) and refactored the styles!